XenAdmin/Wizards/NewSRWizard_Pages/Frontends/LVMoHBA.cs

```diff
@@ -56,12 +56,17 @@ public LVMoHBA()
             InitializeComponent();
         }
 
-        public virtual SR.SRTypes SrType { get { return SR.SRTypes.lvmohba; } }
+        public virtual SR.SRTypes SrType { get { return srProvisioningMethod.Lvm ? SR.SRTypes.lvmohba : SR.SRTypes.gfs2; } }
 
         public virtual bool ShowNicColumn { get { return false; } }
 
         public virtual LvmOhbaSrDescriptor CreateSrDescriptor(FibreChannelDevice device)
         {
+            return SrType == SR.SRTypes.gfs2 ? new Gfs2HbaSrDescriptor(device) : new LvmOhbaSrDescriptor(device, Connection);
+        }
+
+        public virtual LvmOhbaSrDescriptor CreateLvmSrDescriptor(FibreChannelDevice device)
+        {

```

Comment:
There are two blocks missing from this file (compared with #972)
- https://github.com/xenserver/xenadmin/pull/972/files#diff-bb0607ef13f8cd2a2d82e6bad8edc894R102
- https://github.com/xenserver/xenadmin/pull/972/files#diff-bb0607ef13f8cd2a2d82e6bad8edc894R339